vaadin-button-flow-parent/vaadin-button-flow/src/main/java/com/vaadin/flow/component/button/GeneratedVaadinButton.java

```diff
@@ -122,7 +52,10 @@
      * </p>
      *
      * @return the {@code autofocus} property from the webcomponent
+     *
+     * @deprecated since v23.3, will be removed in v24.
      */
+    @Deprecated
```

Comment:
We have discussed yesterday that we want to deprecate all protected methods and constructors, however that has not been applied to this PR.

Reply:
Added in the latest commit